bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,22 +24,12 @@
         {% if not hide_nav_download_button %}
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
-          {% if LANG in ['en-US', 'de', 'fr'] %}
-            {% if LANG == 'de' %}
-              {% set fxa_button_text = 'Firefox-Konto erstellen' %}
-              {% set fxa_link_text = 'Das sind deine Vorteile' %}
-            {% elif LANG == 'fr' %}
-              {% set fxa_button_text = 'Créez un compte Firefox' %}
-              {% set fxa_link_text = 'Découvrez les avantages' %}
-            {% else %}
-              {% set fxa_button_text = 'Get a Firefox Account' %}
-              {% set fxa_link_text = 'Check out the Benefits' %}
-            {% endif %}
+          {% if LANG in ['en-US', 'en-CA', 'en-GB', 'en-ZA', 'de', 'fr'] %}
             <div class="c-navigation-fxa-cta-container">
-              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta">
-                {{ fxa_button_text }}
+              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta" data-alt-href="{{ url('firefox.accounts') }}">
+                {{ _('Get a Firefox Account') }}
               </a>
-              <p class="c-navigation-fxa-cta-small-link"><a data-link-type="nav" data-link-name="Check out the Benefits" href="{{ url('firefox.accounts') }}">{{ fxa_link_text }}</a></p>
+              <p class="c-navigation-fxa-cta-small-link"><a data-link-type="nav" data-link-name="Check out the Benefits" href="{{ url('firefox.accounts') }}">{{ _('Check out the Benefits') }}</a></p>
```

Comment:
If I'm signed into FxA I see two links that go to the same page:

<img width="240" alt="Screenshot 2019-03-12 at 13 41 16" src="https://user-images.githubusercontent.com/400117/54204724-bc9e0300-44cc-11e9-9c3e-66c9ae224557.png">


Should we only show this small link if signed out?

Reply:
I tried that but then the small link is visible for a split second until the JS determines the user state and hides it. And since it also has an effect on the navbar height, showing/hiding that link can make the entire page jump by a few pixels. I decided the redundant link was better, but maybe there's another way around it?

Reply:
Maybe only show it if not already signed in?

Reply:
That's also possible, but then it's hidden and suddenly pops into view once we determine you're signed out and it's just as bad as it being visible and then vanishing if you're signed in.

Something that might help is if we 1) hide it by default, but reserve space for it to avoid the page jitter using `visibility:hidden` rather than `display:none` and 2) add a slight transition so it fades in rather than becoming visible instantly. Then it still draws attention but might feel more deliberate. Worth a try maybe?

Reply:
Good point yeah, the `visibility: hidden` suggestion sounds like a good middle ground. 

But perhaps we're now on the brink of over complicating this just for users who are signed-in, which to be honest are a minority. Maybe showing this link is acceptable enough for either state. If someone has an account, they can still learn more about the benefits they may not know about?